src/Form/Input.php

```diff
@@ -195,4 +207,20 @@ protected function onPreRender(View &$view)
         $view->pattern  = $this->pattern;
         $view->required = $this->required;
     }
+
+    /**
+     * @return DataTransformerInterface
+     */
+    public function getDataTransformator()
+    {
+        return $this->dataTransformator;
+    }
+
+    /**
+     * @param DataTransformerInterface $dataTransformator
+     */
+    public function setDataTransformator($dataTransformator)
```

Comment:
`public function setDataTransformator(DataTransformerInterface $dataTransformator)`

Reply:
Which PSR does the project use? I might have overlooked this information. If it is not written somewhere we should add it